### PR TITLE
First run changes

### PR DESCRIPTION
**Pull request by amanarora380**

### CODE REVIEW

File: src/Routes/index.jsx [hunk 1]
```diff
@@ -3,16 +3,18 @@ import Login from "Pages/Login/index";
 import DashboardWrapper from "Pages/Dashboard/index";
 import GenerateLicenseWrapper from "Pages/License/Generate/index";
 import SearchLicenseWrapper from "Pages/License/Search";
-
 export const LICENSE_SEARCH_PAGE_ROUTE = "/license/search";
 
+import FirstRunWrapper from "../Pages/SystemConfiguration/FirstRunWrapper";
```
Comment (pinak-globallogic) [inline src/Routes/index.jsx:8]: use absolute path

File: src/Pages/SystemConfiguration/FirstRunWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import FirstRunStepper from "./FirstRunStepper";
+import Grid from "@mui/material/Grid";
+import ResponsiveAppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import Container from "@mui/material/Container";
+import logo from "../../Assets/Images/entrust-logo.png";
```
Comment (pinak-globallogic) [inline src/Pages/SystemConfiguration/FirstRunWrapper.jsx:6]: use absolute path

File: src/Pages/SystemConfiguration/FirstRunWrapper.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+const FirstRunWrapper = () => {
+  return (
+    <>
+      <ResponsiveAppBar>
+        <Container>
+          <Toolbar disableGutters>
+            <img src={logo} alt="entrust-logo.png" style={{ height: "40px" }} />
```
Comment (pinak-globallogic) [inline src/Pages/SystemConfiguration/FirstRunWrapper.jsx:14]: use loading=lazy for images for optimization

File: src/Pages/SystemConfiguration/FirstRunStepper.jsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            <Step key={label} {...stepProps}>
+              <StepLabel {...labelProps}>{label}</StepLabel>
+            </Step>
+          );
+        })}
+      </Stepper>
+
+      {getCurrentStepDetails({ activeStep })}
+      <React.Fragment>
+        <Box sx={{ display: "flex", flexDirection: "row", pt: 2 }}>
```
Comment (pinak-globallogic) [inline src/Pages/SystemConfiguration/FirstRunStepper.jsx:90]: Refactor styles with makeStyles for all inline styles. Refer Login component

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: src/Routes/index.jsx (hunk 1 above)
Comment (amanarora380) [reply src/Routes/index.jsx]: done

**Review by pinak-globallogic — APPROVED** [review]

**Pull request merged**